webpack.config.cjs

```diff
@@ -2,6 +2,8 @@ const path = require( 'node:path' );
 const CopyPlugin = require( 'copy-webpack-plugin' );
 const { TsconfigPathsPlugin } = require( 'tsconfig-paths-webpack-plugin' );
 const FileManagerPlugin = require( 'filemanager-webpack-plugin' );
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
We're relying on webpack being installed through `wp-scripts` (because it's a dependency of wp-scripts), but since we rely on it directly, we could consider adding it to our `package.json`.

Reply:
oh, that's why IDE was complaining and I thought it was nuts to say that since webpack works 😆 
Added!